CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,73 @@
+- Add two new fields 'sourcePort' and 'sourceEndPort' in Antrea-native policy API to match traffic initiated from specific ports. ([#4687](https://github.com/antrea-io/antrea/pull/4687), [@Dyanngg])
+- Add a new field 'logLabel' to ACNP and ANP CRDs to specify firewall log label for Audit Logging. ([#4748](https://github.com/antrea-io/antrea/pull/4748), [@qiyueyao])
+- Add a new Controller configuration 'clientCAFile' to allow user to specify client CA. ([#4664](https://github.com/antrea-io/antrea/pull/4664), [@wenyingd])
+- Add support for ExternalIP in AntreaProxy to allow a Service to be accessed from outside the cluster using an external IP address. ([#4866](https://github.com/antrea-io/antrea/pull/4866), [@hongliangl])
+- Add OVS table name as label for 'ovs_flow_count' Prometheus metrics. ([#4893](https://github.com/antrea-io/antrea/pull/4893), [@cr7258])
+- Add WireGuard tunnel mode for Antrea Multi-cluster to support encryption of the traffic between member clusters. ([#4737](https://github.com/antrea-io/antrea/pull/4737) [#4606](https://github.com/antrea-io/antrea/pull/4606), [@hjiajing])
+- Add WireGuard encryption tunnel mode to Multi-cluster user guide. ([#4848](https://github.com/antrea-io/antrea/pull/4848), [@hjiajing])
+- Add support for EndpointSlice API for Multi-cluster Services. When the EndpointSlice API is available for the cluster, EndpointSlice resources of the exported Service, rather than the Endpoints resource, will be watched. For the Pod IP type of Services, EndpointSlice resources will be exported as the Multi-cluster Service's endpoints. ([#4895](https://github.com/antrea-io/antrea/pull/4895), [@luolanzone])
+- Add IP based group association query API and ClientSet for querying the Antrea Group/ClusterGroups. ([#4807](https://github.com/antrea-io/antrea/pull/4807), [@Dyanngg])
+- Add new exporter to FlowAggregator to write flows to local file. ([#4855](https://github.com/antrea-io/antrea/pull/4855), [@antoninbas])
+- Support new OS openEuler 22.03 and add it in tested distributions. ([#4957](https://github.com/antrea-io/antrea/pull/4957), [@ceclinux])
+
+### Changed
+
+- Deprecate Antrea Octant Plugin. ([#4825](https://github.com/antrea-io/antrea/pull/4825), [@antoninbas])
+- Update Open vSwitch version to 2.17.6. ([#4959](https://github.com/antrea-io/antrea/pull/4959), [@tnqn])
+- Update Windows OVS version to 2.16.7. ([#4705](https://github.com/antrea-io/antrea/pull/4705), [@XinShuYang])
+- Upgrade K8s libraries to v0.26.4. ([#4935](https://github.com/antrea-io/antrea/pull/4935), [@heanlan])
+- Bump up whereabouts to v0.6.1. ([#4988](https://github.com/antrea-io/antrea/pull/4988), [@hjiajing])
+- Update Egress API to support multiple Egress IPs and pools. ([#4603](https://github.com/antrea-io/antrea/pull/4603), [@tnqn])
```

Comment:
This is not really supported yet and might confuse users. It could say "Add `status.egressIP`  field for Egress to represent the effective Egress IP"